### PR TITLE
ansible: actions on list of elements

### PR DESCRIPTION
**Pull request by pierrecregut**
Provides actions on lists  of elements that are much faster
than repeating actions on one element at a time.

### CODE REVIEW

File: contrib/ansible/library/skydive_node_list.py [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+        finally:
+            self.factory.client.loop.call_soon(self.stop)
+
+
+def run_module():
+    module_args = dict(
+        analyzer=dict(type='str', default="127.0.0.1:8082"),
+        ssl=dict(type='bool', default=False),
+        insecure=dict(type='bool', default=False),
+        username=dict(type='str', default=""),
```
Comment (EmilienM) [inline contrib/ansible/library/skydive_node_list.py]: shouldn't username & passwords be required?

Comment (pierrecregut) [reply contrib/ansible/library/skydive_node_list.py]: no, if you use the noauth backend for skydive.

File: contrib/ansible/library/skydive_edge_list.py [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+        description:
+            - host of the node
+
+notes:
+  - Using list reduces the communication overhead for creating edges.
+  - Node definitions must be UUIDs. Gremlin queries require a communication
+    roundtrip.
+
+author:
+    - Sylvain Afchain (@safchain)
```
Comment (safchain) [inline contrib/ansible/library/skydive_edge_list.py]: You should maybe change this line :)

File: contrib/ansible/library/skydive_node_list.py [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+        description:
+            - list of nodes
+        required: true
+
+notes:
+  - The default value of seed may be unsufficient to disambiguate nodes.
+  - Using list reduces the communication overhead for creating nodes.
+
+author:
+    - Sylvain Afchain (@safchain)
```
Comment (safchain) [inline contrib/ansible/library/skydive_node_list.py]: ditto

File: contrib/ansible/library/skydive_edge_list.py (hunk 2 above)
Comment (pierrecregut) [reply contrib/ansible/library/skydive_edge_list.py]: I will add a line as secondary author, my changes are minor compared to your code. This is one of the thing I do not like much about this code: there is a lot of boiler plate shared between modules. But it seems to be hard to share code between modules in ansible without putting constraints on the user (https://groups.google.com/forum/#!topic/ansible-devel/Tv_9GXnDJRI).

Comment (safchain) [reply contrib/ansible/library/skydive_edge_list.py]: And it was a such excellent code :) Yes not easy to share code between modules it seems. Maybe we could add helpers into the python lib directly but that is something we can address later. Thanks for the PR.

**Comment by safchain** [comment]
rerun skydive-k8s-tests

**Comment by safchain** [comment]
rerun skydive-k8s-tests

**Pull request merged**